doc/howtos/building-images.rst

```diff
@@ -256,6 +284,52 @@ reconfigure it so that it matches ``ostro-image-swupd-dev``::
     OSTRO_IMAGE_NOSWUPD_EXTRA_FEATURES_append = "${OSTRO_IMAGE_FEATURES_DEV}"
     OSTRO_IMAGE_NOSWUPD_EXTRA_INSTALL_append = "${OSTRO_IMAGE_INSTALL_DEV}"
 
+Adding a Custom Layer in Ostro OS
+---------------------------------
+
+There are two practical options for adding a custom layer into your Ostro OS build.
+The first keeps your custom layer independent from ostro-os:::
+
+   $ git clone <your-custom-layer>
+   $ source oe-init-build-env
+   $ bitbake-layers add-layer $LAYERDIR     # add the custom layer path in OSTRO_LAYERS
+   $ bitbake-layers show-layers             # verify bitbake sees the layer
+
+A second approach is to add your custom layer by pulling it into the Ostro build with the ``combo-layer`` script::
+
+   $ git clone https://github.com/ostroproject/ostro-os.git     # or clone using git: if you've got your ssh keys setup
+   $ cd ostro-os
+   $ $EDITOR conf/combo-layer.conf        # add the layer repo details in the combo-layer.conf file
+   $ $EDITOR conf/combo-layer-local.conf  # and match those changes and repo local path
```

Comment:
Just realised there are more complete instructions on how to use combo-layer in the top-level README here: https://github.com/ostroproject/ostro-os#updating-the-repository

I'd suggest we park that for now and make it an enhancement for post 1.0.